lib/matplotlib/tests/test_sphinxext.py

```diff
@@ -97,10 +97,33 @@ def plot_directive_file(num):
     assert filecmp.cmp(range_6, plot_file(5))
 
 
-def build_sphinx_html(source_dir, doctree_dir, html_dir):
+def test_plot_html_show_source_link(tmpdir):
+    source_dir = Path(tmpdir) / 'src'
+    source_dir.mkdir()
+    shutil.copyfile(Path(__file__).parent / 'tinypages' / "conf.py", source_dir / "conf.py")
+    shutil.copytree(Path(__file__).parent / 'tinypages' / "_static", source_dir / "_static")
```

Comment:
Can combine this, but at the very least, use consistent quotes on a line.
```suggestion
    shutil.copyfile(Path(__file__).parent / 'tinypages/conf.py', source_dir)
    shutil.copytree(Path(__file__).parent / 'tinypages/_static', source_dir)
```